param_sweeps/driver.py

```diff
@@ -44,7 +44,7 @@ def from_input_file(cls, ifile: InputFile):
         cls_fields = list(signature(cls).parameters)
         base_params, app_params = {}, {}
 
-        for param, value in ifile.data.items():
+        for param, value in ifile.plain_data.items():
```

Comment:
using ifile.data when it contains a Workspace leads to an exception as it cannot be serialized
(unclear to me how it has been working  before)

Reply:
Not sure i am following - ifile.data is a dict, which shouldn't require elements to be seriliazable for the loop over items?

Reply:
thanks for insisting ! That change is indeed not necessary 😄 

I have tried to reproduce and tracked down my initial problem:
it was coming from a test in param-sweep where I gave a string instead of a Workspace. Then i failed in the validator, so I have changed geoh5 to instantiate the workspace from the string ion the fly so that it passes the validator, but then the serialization was failing, etc, etc...
Anyway, all gone. Also in geoh5py https://github.com/MiraGeoscience/geoh5py/pull/347